package-system/pyside2/build_linux.sh

```diff
@@ -27,7 +27,7 @@ then
 fi
 
 # Set the dependent clang compiler for the build script to use
-LLVM_INSTALL_DIR=$TEMP_FOLDER/libclang-release_130-based-linux-Ubuntu20.04-gcc9.3-x86_64/libclang
+LLVM_INSTALL_DIR=/usr/lib/llvm-6.0
```

Comment:
Can we remove the `additional_download_packages` for linux now if it's going to use `/usr/lib/llvm-6.0`?

Reply:
Good catch, this is a mistake and needs to be reverted